src/job/job.py

```diff
@@ -840,7 +840,6 @@ def run_docker_container(self, image_name):
         try:
             c.header("Run container", show=True)
             c.execute(cmd, show=True, show_cmd=False)
-            c.execute(("docker", "commit", container_name, image_name))
```

Comment:
The docker commit has to happen here, because else we would only do the commit in case of an error.
So basically we should move the if from below to here.